### PR TITLE
filter only verified companies

### PR DESCRIPTION
**Pull request by petarblazevski**

### CODE REVIEW

File: company/getAll.js [hunk 1]
```diff
@@ -15,7 +15,11 @@ export const main = async () => {
   try {
     do {
       items = await dynamoDbLib.call('scan', params);
-      items.Items.forEach(item => companies.push(item));
+      items.Items.forEach(item => {
+        if (item.isVerified && item.stripeUserId.length > 0) {
+          companies.push(item);
+        }
+      });
```
Comment (IAfanasovMob) [inline company/getAll.js:22]:
```
companies =  items.Items.filter(item => item.isVerified && item.stripeUserId.length);
```

Comment (petarblazevski) [reply company/getAll.js:22]: I am not sure why this is here `params.ExclusiveStartKey = items.LastEvaluatedKey;`. That's why I haven't used `filter`. Maybe @cezaryskura can give some input

Comment (ghost) [reply company/getAll.js:22]: https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html

Comment (petarblazevski) [reply company/getAll.js:22]: Pagination I guess. We might need to change it all the places

Comment (IAfanasovMob) [reply company/getAll.js:22]: sorry, I missed that one
make sense

**Review by IAfanasovMob — APPROVED** [review]
seems like it needs rebase and squash

**Pull request merged**